tests/test_poincare_ball.py

```diff
@@ -23,7 +23,7 @@ def setUp(self):
         self.hyperboloid_manifold = Hyperboloid(2)
         self.hyperboloid_metric = self.hyperboloid_manifold.metric
 
-    @geomstats.tests.np_and_pytorch_only
+    @geomstats.tests.np_only
```

Comment:
What happened for the pytorch test here?

Reply:
My bad, it should works with pytorch.